script.js

```diff
@@ -2,10 +2,51 @@ let navToggle = document.getElementsByClassName("navToggle")[0]
 let navLinks = document.getElementsByClassName("navLinks")[0]
 
 navToggle.addEventListener("click", menuDown);
-function menuDown(){
-    if(navLinks.style.display === "flex"){
-        navLinks.style.display="none"
-    }else{
-        navLinks.style.display="flex"
+function menuDown() {
+    if (navLinks.style.display === "flex") {
+        navLinks.style.display = "none"
+    } else {
+        navLinks.style.display = "flex"
     }
 }
+
+let navbar = document.getElementsByClassName("navbar")[0]
+
+window.onscroll = function () {
+    let verticalSpace = 200
```

Comment:
I think it's better to use`const` here since we're not planning to change it. Also usually such type of config variables are named using snake case + upper case. Example `VERTICAL_SPACE`